src/meta/src/hummock/manager/commit_epoch.rs

```diff
@@ -289,6 +289,24 @@ impl HummockManager {
                 .values()
                 .map(|g| (g.group_id, g.parent_group_id))
                 .collect();
+            let time_travel_tables_to_commit =
```

Comment:
You are able to get the compaction group id of a table_id in the local variable `table_compaction_group_mapping`. It includes compaction group id of both the newly added table_ids and the existing ones.

Reply:
Fixed